classes/autoptimizeConfig.php

```diff
@@ -194,7 +206,7 @@ public function show() {
 <td><?php
 	$AOstatArr=autoptimizeCache::stats(); 
 	$AOcacheSize=round($AOstatArr[1]/1024);
-	echo $AOstatArr[0].__(' files, totalling ','autoptimize').$AOcacheSize.__(' Kbytes (calculated at ','autoptimize').date("H:i e", $AOstatArr[2]).')';
+	echo $AOstatArr[0].' '.__(' files, totalling ','autoptimize').' '.$AOcacheSize.' '.__(' Kbytes (calculated at ','autoptimize').' '.date("H:i e", $AOstatArr[2]).')';
```

Comment:
Add this because I thought that when someone performs the translations, it could happen that sometimes those spaces are not easily detected and therefore could be omitted for the translator or considered as error. I suggested this change just to contemplate such situations since successive spaces in the HTML code are not considered when rendering the page and are only 4 bytes. :)